projects/first-web/public/index.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta content="width=device-width,initial-scale=1" name="viewport">
+    <title>Trivial web page</title>
+</head>
+<body>
+  <h1>Ellis' Web Page!</h1>
+  <p><b>--------------------------------</b></p>
+  <p1> <a href="https://elliswaterman-symmetrical-telegram-9rww9gr6jg4hp7rx-3000.preview.app.github.dev/">Quiz</a>
```

Comment:
You can make this more robust by not including the server part of the URL:

```suggestion
  <p1> <a href="/">Quiz</a>
```

The missing server will default to the server the current page was served from which is what you want. That way this HTML will work wherever it's served from. The way this is, if you created a new codespace it wouldn't work. However in this case I think maybe you want:

```javascript
  <p1> <a href="/quiz.html">Quiz</a>
```